src/rt-app.c

```diff
@@ -1010,12 +1010,18 @@ static void set_thread_priority(thread_data_t *data, sched_data_t *sched_data)
 			sa_params.sched_flags |= SCHED_FLAG_UTIL_CLAMP_MIN;
 			log_notice("[%d] setting util_min=%d",
 				   data->ind, sched_data->util_min);
+			log_ftrace(ft_data.marker_fd, FTRACE_ATTRS,
+				   "rtapp_attrs: event=uclamp util_min=%d",
```

Comment:
Same here. Not sure if using the same event name will produce something that will be hard to distinguish in pandas when converted to a data frame by Lisa. I'm probably wrong but worth asking :-)

Reply:
The same response/question above applies.